tests/testthat/test-model-options.R

```diff
@@ -2,18 +2,18 @@ context("model-options")
 
 test_that("can build JSON from template", {
   json <- build_json("test <+param1+>", list(param1 = scalar("test string")))
-  expect_equal(json, 'test "test string"')
+  expect_equivalent(json, 'test "test string"')
```

Comment:
What attributes mismatch here now?

Reply:
Glue has changed and now outputs from `glue::glue` have class `glue`

Reply:
that's so weird. Why would you want that? `sprintf()` does not return objects with class "sprintf"...